spec/features/studies/view_study_properties_spec.rb

```diff
@@ -54,6 +54,10 @@
 
     it 'displays HuMFre approval number for Not Applicable (Contact Datasharing) data release strategy' do
       study.study_metadata.data_release_strategy = Study::DATA_RELEASE_STRATEGY_NOT_APPLICABLE
+      study.study_metadata.data_release_timing = Study::DATA_RELEASE_TIMING_NEVER
+      study.study_metadata.data_release_prevention_reason = Study::DATA_RELEASE_PREVENTION_REASONS[0]
+      study.study_metadata.data_release_prevention_approval = Study::YES
+      study.study_metadata.data_release_prevention_reason_comment = 'comment'
```

Comment:
Unless the feature is enabled for the test, the lines has no effect. I guess they are for completeness.

Reply:
Good point... I added the feature flag after making this change to the test, so it would have been erroring. I think it's probably OK to leave it like this because this is not testing my feature, it's just something that would have broken when I enabled the feature flag.

Reply:
If the feature (validation) is enabled, the lines will make the test pass; without the lines the test will fail. It documents what the fields must be together. Keeping them is OK.